dashboards-notebooks/public/components/paragraph_components/paragraphs.tsx

```diff
@@ -437,7 +436,29 @@ export const Paragraphs = forwardRef((props: ParagraphProps, ref) => {
     }
   };
 
-  const paragraphLabel = "Specify the input language on the first line using %[language type]. Supported languages include markdown, SQL and PPL"
+  const sqlIcon = (
+    <EuiLink href="https://opendistro.github.io/for-elasticsearch-docs/docs/sql/" target="_blank"> SQL <EuiIcon type="popout" size="s"/> </EuiLink>
```

Comment:
can we put documentation links here? https://github.com/opensearch-project/dashboards-notebooks/blob/dev/dashboards-notebooks/common/index.ts#L36

Reply:
Yes forgot to use opensearch doc links, will amend